script/up

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 set -euo pipefail
-CLUSTER="example"
+CLUSTER="kb-example"
 SERVICE="doks-example"
-CONTEXT="do-nyc1-${CLUSTER}"
-doctl k8s cluster create ${CLUSTER}
+REGION="lon1"
```

Comment:
Would you mind moving this back to `nyc1`, again to be consistent with our README instructions? It's also the default datacenter chosen when you create DOKS clusters via `doctl`.

If you think this should be configurable, feel free to change it to

```shell
REGION=${REGION:-nyc1}
```

That way, you can invoke the script like

```shell
REGION=lon1 ./script/up
```

and override the default region.